tools/install_rust.sh

```diff
@@ -0,0 +1,5 @@
+if [ ! -d rust-installer ]; then
+    mkdir rust-installer
+    wget https://sh.rustup.rs -O rust-installer/rustup.sh
+    sh rust-installer/rustup.sh -y
+fi
```

Comment:
don't know how important it is, but does this script want a `#!/bin/sh`?

Reply:
It doesn't hurt, I'll add this

Reply:
Added in: https://github.com/Qiskit/qiskit-terra/pull/7658/commits/eaf92f2199b4a14472ad2b8a6155a29c1cc5b782